docs/get-started.md

```diff
@@ -183,14 +183,12 @@ Once fully removed, the backup is no longer visible when you run:
 velero backup get BACKUP_NAME
 ```
 
-If you want to uninstall Velero but preserve the backup data in object storage and persistent volume
-snapshots, it is safe to remove the `velero` namespace and everything else created for this
-example:
+To completely uninstall Velero, minio, and the nginx example app from your Kubernetes cluster:
 
 ```
-kubectl delete -f config/common/
-kubectl delete -f config/minio/
-kubectl delete -f config/nginx-app/base.yaml
+kubectl delete namespace/velero clusterrolebinding/velero
+kubectl delete crds -l component=velero
+kubectl delete -f examples/nginx-app/base.yaml
```

Comment:
Looks like this is missing minio now

Reply:
nvm, it's in the velero namespace.

Reply:
yeah, this section didn't make sense before -- the text said you'd keep your backups in object storage, but then the delete script deleted minio!